Pod/Classes/IPStringBackedDecimalValue.h

```diff
@@ -0,0 +1,24 @@
+//
+//  IPStringBackedDecimalValue.h
+//  Pods
+//
+//  Created by Nicholas Servidio on 10/26/15.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+@interface IPStringBackedDecimalValue : NSObject
+
+@property (strong, nonatomic, readonly) NSMutableString *currentValue;
+@property (nonatomic) NSUInteger maxNumberOfWholeNumberDigits;
+@property (nonatomic) NSUInteger maxNumberOfDecimalDigits;
+
+- (NSString *)currentValueFormattedAsPrice;
```

Comment:
Maybe not for this pr, but imho formatters is something that we want to be flexible about. Also, formatters are likely to be view specific, not tied to the model.